scripts/signer/signer.ps1

```diff
@@ -0,0 +1,87 @@
+#
+#  Copyright (c) Contributors to the Open 3D Engine Project
+# 
+#  SPDX-License-Identifier: Apache-2.0 OR MIT
+# 
+# 
+
+param (
+    [String[]] $exePath,
+    [String[]] $packagePath,
+    [String[]] $bootstrapPath,
+    [String[]] $certificate
+)
+
+# Get prerequisites, certs, and paths ready
+$tempPath = "C:\\temp"
+$certThumbprint = Get-ChildItem -Path Cert:LocalMachine\MY -ErrorAction Stop | Select-Object -ExpandProperty Thumbprint # Grab first certificate from local machine store
+
+if ($certificate) {
+    Write-Output "Checking certificate thumbprint $certificate"
+    Get-ChildItem -Path Cert:LocalMachine\MY -ErrorAction SilentlyContinue | Where-Object {$_.Thumbprint -eq $certificate} # Prints certificate Thumbprint and Subject if found
+    if($?) {
+        $certThumbprint = $certificate
+    }
+    else {
+        Write-Error "$certificate thumbprint not found, using $certThumbprint thumbprint instead"
+    }
+}
+
+Try { 
+    $signtoolPath = Resolve-Path "C:\Program Files*\Windows Kits\10\bin\*\x64\signtool.exe" -ErrorAction Stop | Select-Object -Last 1 -ExpandProperty Path
+    $insigniaPath = Resolve-Path "C:\Program Files*\WiX*\bin\insignia.exe" -ErrorAction Stop | Select-Object -Last 1 -ExpandProperty Path
+}
+Catch {
+    Write-Error "Signtool or Wix insignia not found! Exiting."
+}
+
+function Write-Signiture {
+    param (
+        $signtool,
+        $thumbprint,
+        $filename
+    )
+    Try {
+        & $signtool sign /tr http://timestamp.digicert.com /td sha256 /fd sha256 /sha1 $thumbprint /sm $filename
+        & $signtool verify /pa /v $filename
+        }
+    Catch {
+        Write-Error "Failed to sign $filename!"
+    }
+}
+
+# Looping through each path insteaad of globbing to prevent hitting maximum command string length limit
+if ($exePath) {
+    Write-Output "### Signing EXE files ###"
+    $files = @(Get-ChildItem $exePath -Recurse *.exe | % { $_.FullName })
```

Comment:
This follows the same business logic from LY 1.x, in that only the exe files were signed. This is also only directed at the bin folder from the o3de output directory, instead of a wider range. The bootstrapper already fingerprints all binaries and will fail to install if anything has changed (hence why the bootstrapper, cab, and msi is also signed).